CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.6)
+cmake_minimum_required(VERSION 3.16)
```

Comment:
We should just not forget to check if all build machines have an up to date version installed.

Reply:
I think especially the macOS build needs an update.

Reply:
so this one is triggering a warning when trying to find the KF5 libraries we use to integrate with KDE's file manager dolphin